.env-example

```diff
@@ -27,7 +26,7 @@ BPA_DID_PREFIX=did:sov:iil:
 
 # The Ledger Explorer
 LEDGER_URL=https://indy-test.bosch-digital.de
-BPA_LEDGER_BROWSER=${LEDGER_URL}
+BPA_LEDGER_BROWSER=https://indy-test.bosch-digital.de
```

Comment:
Wouldn't it make more sense to add the version to the compose file again and add a compose version requirement to the documentation?

Reply:
This a general suggestion, that's not related to the specific code lines above, isn't ?

I'd be fine with both approaches. However, I don't think the changes I made here would be too problematic and it might prevent people that don't read the documentation in detail to get errors they don't understand. So I slightly favor this approach.

Reply:
Maybe its even better to simply change this line in the compose file:

`BPA_LEDGER_BROWSER: ${BPA_LEDGER_BROWSER}`

to

`BPA_LEDGER_BROWSER: ${LEDGER_URL}`

Reply:
The reason why I like to keep this separate in the env file is the following.

LEDGER_URL is used to get the genesis file and does not need to be bundled with a ledger explorer/browser

BPA_LEDGER_BROWSER is the explorer/browser the BPA uses to get schemas/credential definitions from the ledger.

These two functionalities are pretty different.